modules/persistent-storage-csi-efs-static-pv.adoc

```diff
@@ -6,13 +6,13 @@
 
 :_content-type: PROCEDURE
 [id="efs-create-static-pv_{context}"]
-= Creating static PVs with AWS EFS
+= Creating static PVs with Amazon EFS
 
-It is possible to use an AWS EFS volume as a single PV without any dynamic provisioning. The whole volume is mounted to pods.
+It is possible to use an Amazon EFS volume as a single PV without any dynamic provisioning. The whole volume is mounted to pods.
 
 .Prerequisites
 
-* You have created AWS EFS volumes.
+* You have created Amazon EFS volumes.
```

Comment:
Revoked. The service name seems to be associated with object.

Reply:
Updated, as per the https://github.com/openshift/openshift-docs/pull/59650#discussion_r1290144425